standalone-e2e/standalone-binary.test.js

```diff
@@ -0,0 +1,80 @@
+const util = require('util')
```

Comment:
Would that be OK to write this file in typescript to have all our code in the same language as much as possible?